frontend/mr-admin-flate/src/components/avtaler/AvtaleListe.tsx

```diff
@@ -0,0 +1,48 @@
+import { Alert, BodyShort } from "@navikt/ds-react";
+import { Laster } from "../laster/Laster";
+import styles from "../tiltaksgjennomforinger/TiltaksgjennomforingerListe.module.scss";
+import { AvtaleFilter } from "../../api/atoms";
+import { Avtale } from "mulighetsrommet-api-client";
+import { ReactNode } from "react";
+import { useAvtaler } from "../../api/avtaler/useAvtaler";
+import { AvtalestatusTag } from "../statuselementer/AvtalestatusTag";
+
+interface Props {
+  filter: Partial<AvtaleFilter>;
+  action: (avtale: Avtale) => ReactNode;
+}
+
+export function AvtaleListe(props: Props) {
+  const { data, isError, isPending } = useAvtaler(props.filter);
+
+  if (isError) {
+    return <Alert variant="error">Vi hadde problemer med å hente avtaler</Alert>;
+  }
+
+  if (isPending) {
+    return <Laster size="xlarge" tekst="Laster avtaler..." />;
+  }
+
+  const avtaler = data.data;
+
+  return (
+    <div className={styles.gjennomforingsliste_container}>
```

Comment:
Bærer en del preg av copy-paste fra gjennomføringer til avtale, det samme gjelder en del andre komponenter i disse to modulene.

Planlegger å ta en egen runde å generalisere litt på tvers av avtaler/gjennomføringer.